lib/rex/post/meterpreter/extensions/stdapi/sys/process.rb

```diff
@@ -395,7 +395,10 @@ def to_table(opts={})
     # ppid.  Cut columns from the list if they aren't there.  It is conceivable
     # that processes might have different columns, but for now assume that the
     # first one is representative.
-    cols.delete_if { |c| !( first.has_key?(c.downcase) ) or first[c.downcase].nil? }
+    cols.delete_if do |c|
+      !(any? {|r| r.has_key?(c.downcase)}) or
```

Comment:
`||` over `or`, per the style guide. Also, it's not in the style guide but I think I prefer `none?` over `!(any? ...)`.